pyvelociraptor/wrappers.py

```diff
@@ -0,0 +1,146 @@
+
+
+from velo_pandas import DataFrameQuery as run
+import pandas as pd
+import time
+
+pd.set_option('display.max_colwidth', None)
+pd.set_option('display.max_columns', None)
+pd.set_option('display.max_rows', None)
+
+# helper functions
+def getdf(vql):
+    return pd.DataFrame(run(vql))
+
+def getparams(d):
```

Comment:
Not needed - just expand parameters directly into the DataFrameQuery using `**kwargs`

You might need to convert them all to string if you want to allow users to specify any types.